tests/test_analysis.py

```diff
@@ -1905,6 +1906,49 @@ def test_calc_confidence_interval_overlap(self):
         overlap = aopy.analysis.calc_confidence_interval_overlap(CI1,CI2)
         self.assertEqual(overlap,(20-17)/(20-10))
         
+
+class TestCalcMaxCohAndLags(unittest.TestCase):
```

Comment:
Can you make the name of the test class "test_{function_name_to_be_tested}"? This way it is easy to identify which set of tests belongs to which function.